apps/etl/transform/sources/pdc.py

```diff
@@ -20,24 +17,36 @@ class PDCTransformHandler(BaseTransformerHandler[PDCTransformer, PDCDataSource])
     transformer_schema = PDCDataSource
 
     @classmethod
-    def get_schema_data(cls, extraction_obj: ExtractionData, geo_json_obj: ExtractionData):  # type: ignore[reportIncompatibleMethodOverride]
-        source_url = extraction_obj.url
+    def get_schema_data(cls, extraction_obj: ExtractionData):
+        metadata: dict | None = extraction_obj.metadata
+        if not metadata:
+            raise Exception("Metadata is not defined")
+        input_metadata = metadata.get("input", {})
+
+        from apps.etl.extraction.sources.pdc.extract import PdcExposureInputMetadata
+
+        input_metadata = PdcExposureInputMetadata(**input_metadata)
```

Comment:
Why generate metadata again if we can get it from ext_ob.metadata.

Reply:
We are not generating the metadata. We are just validating the metdata while reading from database.